package.json

```diff
@@ -3,15 +3,17 @@
   "license": "AGPL-3.0-only",
   "version": "0.1.0",
   "scripts": {
-    "prepare": "npx husky install",
+    "prepare": "husky install",
```

Comment:
I needed to add npx to work in windows i believe / I added it because some error was happening maybe was not windows, need to recheck

Reply:
I see. You can add it back if necessary.